crates/cargo-wdk/src/actions/build/package_task.rs

```diff
@@ -313,9 +314,19 @@ impl<'a> PackageTask<'a> {
             &arch,
             "-c",
             &cat_file_path,
-            "-v",
-            "*",
         ];
+
+        match std::env::var(STAMPINF_VERSION_ENV_VAR) {
+            Ok(version) if !version.trim().is_empty() => {
+                // When STAMPINF_VERSION is set we intentionally omit -v so stampinf reads it
+                // and populates DriverVer.
+                info!("Using {STAMPINF_VERSION_ENV_VAR} env var to set DriverVer: {version}");
```

Comment:
Did you sanity test this behavior on the latest commit while compiling a driver with the env var set?